docs/specs/slots.md

```diff
@@ -0,0 +1,27 @@
+# Slots
+
+Slots are used to replace property values during runtime. Properties that can be replaced will have an `sid` attribute that will correspond to the slot key entry. Players that support slots must also expose an API that applications can use to replace values during runtime.
+
+<h3 id="color-slot">Color Slot</h3>
+
+{schema_object:slots/color-slot}
+
+<h3 id="point-slot">Point Slot</h3>
+
+{schema_object:slots/point-slot}
+
+<h3 id="scale-slot">Scale Slot</h3>
+
+{schema_object:slots/scale-slot}
+
+<h3 id="float-slot">Float Slot</h3>
+
+{schema_object:slots/float-slot}
+
+<h3 id="asset-slot">Asset Slot</h3>
+
+{schema_object:slots/asset-slot}
+
+<h3 id="undefined-slot">Undefined Slot</h3>
```

Comment:
I was on the fence on this as well. My thought was that since it could be in the JSON, someone may want to lookup what it means in which case this may be useful.

Reply:
Obsolete- Typing info removed from 1.0 slot spec